api/src/pcapi/core/educational/models.py

```diff
@@ -92,6 +92,12 @@ class CollectiveBookingCancellationReasons(enum.Enum):
     PUBLIC_API = "PUBLIC_API"
     FINANCE_INCIDENT = "FINANCE_INCIDENT"
     BACKOFFICE = "BACKOFFICE"
+    BACKOFFICE_EVENT_CANCELLED = "BACKOFFICE_EVENT_CANCELLED"
+    BACKOFFICE_OVERBOOKING = "BACKOFFICE_OVERBOOKING"
+    BACKOFFICE_BENEFICIARY_REQUEST = "BACKOFFICE_BENEFICIARY_REQUEST"
+    BACKOFFICE_OFFER_MODIFIED = "BACKOFFICE_OFFER_MODIFIED"
+    BACKOFFICE_OFFER_WITH_WRONG_INFORMATIONS = "BACKOFFICE_OFFER_WITH_WRONG_INFORMATIONS"
```

Comment:
_information_ est invariable en anglais :
```suggestion
    BACKOFFICE_OFFER_WITH_WRONG_INFORMATION = "BACKOFFICE_OFFER_WITH_WRONG_INFORMATION"
```